acoustid/future/fpindex/client.py

```
@@ -0,0 +1,278 @@
+    async def index_exists(self, index_name: str) -> bool:
+        """Check if index exists
+
+        Args:
+            index_name: Name of the index
+
+        Returns:
+            True if index exists, False otherwise
+        """
+        try:
+            status, _ = await self._request(
+                "HEAD", f"/{index_name}", expected_status=[200, 404]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)